README.md

```diff
@@ -63,7 +94,11 @@ An example of three clusters configured to use with Submariner would look like t
 
 # Installation
 
-## Setup
+Submariner installation can be done manually using helm charts or by using its operator. Both of the methods are described here.
```

Comment:
non-native here, so probably wrong "Both of the methods" -> "Both methods"?

Reply:
Done